re-former/app/views/users/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<h1> testing </h1>
+<!--<form action="/users" method="post" accept-charset="UTF-8">
+#<input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+#<label for="email">Email :</label> 
+#  <input type="email" name="user[email]"><br>
+#<label for="username">username :</label> 
+#  <input type="text" name="user[username]"><br>
+#<label for="password">password :</label> 
+#  <input type="password" name="user[password]"><br>
+#  <input type="submit" value="Submit">
+#</form>
+-->
+
+
+<%#= form_tag('/users', method: "post") do %>
+<%#= label_tag('email     : ') %>
+<%#= text_field_tag( :email) %>
+<%#= label_tag('User name : ') %>
+<%#= text_field_tag( :username) %>
+<%#= label_tag('password  : ') %>
+<%#= password_field_tag( :password) %>
+<%#= submit_tag('submit ') %>
+<%#= end %>
+
+<%= form_for @user, url: {action: "create"}  do |f| %>
+  Email  : <%= f.email_field :email , placeholder: 'example@example.com'%><br />
```

Comment:
you don't need to explicitly type Email: you can use `f.label :email`,`f.label :username`,`f.label :password`

Reply:
Changed the code accordingly.